apierror.h

```diff
@@ -68,6 +70,6 @@
 /*! \brief Helper method to get a string representation of an API error code
  * @param[in] error The API error code
  * @returns A string representation of the error code */
-const char *janus_get_api_error(int error);
+shared const char *janus_get_api_error(int error);
```

Comment:
Ah thanks the clarification, I hadn't arrived at that point in my review yet, still looking at the first part. I agree that something like you suggest would avoid ambiguities, and potential issues with dependencies that define the same thing themselves.